File: source/core/hiddenapi/src/main/java/android/os/ServiceManagerHidden.java
package android.os;

/**
 * @see <a href="https://cs.android.com/android/platform/superproject/+/android-8.0.0_r51:frameworks/base/core/java/android/os/ServiceManager.java">ServiceManager.java</a>
 */
public class ServiceManagerHidden {
    public static IBinder getService(String name) {
        throw new RuntimeException("Stub!");
    }
}

File: libhiddenapi/src/main/java/android/os/ServiceManagerHidden.kt
package android.os

import android.annotation.SuppressLint
import org.lsposed.hiddenapibypass.HiddenApiBypass

@SuppressLint("PrivateApi")
object ServiceManagerHidden {
    fun getService(name: String): IBinder {
        return HiddenApiBypass.invoke(Class.forName("android.os.ServiceManager"), null, "getService", name) as IBinder
    }
}
